### DISCUSSION TITLE
Is bumping linting tool version open to new contributors?

### DISCUSSION THREAD

**Discussion by tqa236** [bokeh/bokeh#13681] [Build / CI]
Hello, 

Is bumping linting tool versions open to new contributors or it's better to leave it to a maintainer to handle that?

Thanks!

**Comment by bryevdv** [comment]
Hi @tqa236 apologies this discussion slipped through the cracks. Bumping dependencies currently requires updating several different test env files. If you are able to test things out locally and they work, and you want to update all the various conda env files, feel free to submit a PR. But just be aware anything build/CI related may have a higher chance of working or having other issues once pushed to GH.
